ext/sockets/sockaddr_conv.c

```diff
@@ -137,8 +137,7 @@ int php_set_inet46_addr(php_sockaddr_storage *ss, socklen_t *ss_len, zend_string
 	}
 #endif
 	else {
-		php_error_docref(NULL, E_WARNING,
-			"IP address used in the context of an unexpected type of socket");
+		zend_type_error("IP address used in the context of an unexpected type of socket");
```

Comment:
Is this really type error? Isn't it more a value error?

Reply:
yeah.. I guess I took the type meaning a bit too far here